packages/react-server/core/util/LABString.js

```diff
@@ -1,11 +1,26 @@
-// Careful: We've got a local patch here!
-//
-// We've added support for a `crossOrigin` parameter.
-//
+const path = require("path");
+const {readFileSync} = require("fs");
+
+var lab = readFileSync(path.join(__dirname, "LAB.src.js"), "utf-8");
```

Comment:
Yep, this module is only used in the render middleware.  Its output is a string of JS that is put into an inline script tag.